cloudwash/providers/ec2.py

```diff
@@ -4,8 +4,32 @@
 from cloudwash.utils import dry_data
 from cloudwash.utils import echo_dry
 from cloudwash.utils import total_running_time
+from cloudwash.utils import discovery_timestamp
 from cloudwash.client import compute_client
+import wget
+import tarfile
+import subprocess
+from os.path import exists
+import os
 
+def delete_ocp(ocp_name):
```

Comment:
This entire function should go into https://github.com/RedHatQE/wrapanapi as its the cloudwash dependency to play with resources.